test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/Commands/Package/Update/PackageTests.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.CommandLine;
+using FluentAssertions;
+using NuGet.Versioning;
+using Xunit;
+
+using Pkg = NuGet.CommandLine.XPlat.Commands.Package.Update.Package;
+
+namespace NuGet.CommandLine.Xplat.Tests.Commands.Package.Update
+{
+    public class PackageTests
+    {
+        private RootCommand _command;
+        private Argument<List<Pkg>> _packagesArgument;
+
+        public PackageTests()
+        {
+            _command = new RootCommand();
+
+            _packagesArgument = new Argument<List<Pkg>>("packages")
+            {
+                Arity = ArgumentArity.ZeroOrMore,
+                CustomParser = Pkg.Parse
+            };
+            _command.Arguments.Add(_packagesArgument);
+        }
+
+        [Fact]
+        public void Parse_OnePackage_ReturnsListOfOne()
```

Comment:
nit: my understanding of test names is that the first section should be the Act. So here I'd expect something like this example,
```suggestion
        public void GetValue_WhenOnlyPackageIdProvided_ReturnsPackageIdAndNullVersionRange()
```

Reply:
https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#unit-test-method-naming

It says "what is being tested", which in this case is the `Pakage.Parse` method. But I don't know how to create instances of System.CommandLine's `ArgumentResult`, and frankly I'm not interested in trying to learn because I have more confidence in `Command.Parse` being a stable API, plus I think it's fast enough, so I think testing `Package.Parse` indirectly this way is good enough.

But the class is `PackageTests`, and the `Package` class doesn't have a `GetValue` method, so it doesn't feel right to me to be listed as "what is being tested".  I'm not trying to test System.CommandLine, I assume they have their own tests, I'm trying to test Package.Parse.